qa-tests-backend/src/test/groovy/ProcessBaselinesTest.groovy

```diff
@@ -122,8 +122,19 @@ class ProcessBaselinesTest extends BaseSpecification {
                     (baseline.key.containerName.equalsIgnoreCase(containerName)))
         assert baseline.elementsList.find { it.element.processName == processName } != null
 
-        // Need to sleep so the baseline has time to lock
-        sleep 70000
+        // wait for baseline to come out of observation
+        baseline = evaluateWithRetry(30, 3) {
+            def tmpBaseline = ProcessBaselineService.getProcessBaseline(clusterId, deployment, containerName)
+            def now = System.currentTimeSeconds()
+            if (tmpBaseline.getStackRoxLockedTimestamp().getSeconds() > now) {
+                throw new RuntimeException(
+                    "Baseline ${deployment} is not out of observation yet. Baseline is ${tmpBaseline}.  Current time ${now}"
```

Comment:
Maybe we could rephrase for easier reading? 
_not out of observation yet_ = _still in the observation window_